hail/src/main/scala/is/hail/expr/ir/Copy.scala

```diff
@@ -205,6 +205,9 @@ object Copy {
       case BlockMatrixWrite(_, path, overwrite, forceRowMajor, stageLocally) =>
         val IndexedSeq(newChild: BlockMatrixIR) = newChildren
         BlockMatrixWrite(newChild, path, overwrite, forceRowMajor, stageLocally)
+      case CollectDistributedArray(_, _, ctxName, globalsName, _) =>
```

Comment:
`globalsName` or `globalName`? There's a mix of singular/plural that's a bit confusing to me.

Reply:
I think the intention was to use `global` (although it's always been inconsistent), but `global` is a reserved word in Python, so the Python interface uses `globals`.  I think the two options are to use `globals` everywhere, or to use `globals` in the Python interface only and `global` everywhere else.  I think @tpoterba will have opinions on this. 
 Given the amount of inconsistency, I wouldn't sweat it here, but we should make this a separate cleanup task.

Reply:
This is totally inconsistent within this PR, though (which is what I think Daniel was pointing out), so I'll change to be consistent. (These also don't necessarily represent Table/MatrixTable globals (although Table globals will presumably be a subset of these globals), so I'm wondering if that's an argument for "make the names the same" or "name them something totally different so they're not confusing")

Reply:
I feel like if they're not the same then totally different is the way to go, but ya that's a sweeping change.